packages/calcite-components/src/components/text-area/text-area.tsx

```diff
@@ -311,9 +311,9 @@ export class TextArea
           <textarea
             aria-describedby={this.guid}
             aria-errormessage={IDS.validationMessage}
-            aria-invalid={
-              this.status === "invalid" || toAriaBoolean(this.isCharacterLimitExceeded())
-            }
+            aria-invalid={toAriaBoolean(
+              this.status === "invalid" || this.isCharacterLimitExceeded(),
```

Comment:
nitpick: is the comma needed here?

Reply:
The comma is being added by Prettier in vs code

Reply:
Ok 👍